.circleci/config.yml

```diff
@@ -187,3 +222,14 @@ workflows:
       - run-tests:
           run-unit-tests: false
           context: vscode-test
+
+  cut-release:
```

Comment:
Scheduled job that runs every monday at 7pm from the develop branch. This is a cool tool to verify the cron setting https://crontab.guru/